blog/2017-05-31-devstudio-survey.adoc

```diff
@@ -0,0 +1,18 @@
+= Are you a Developer User Experience lead ?
+:page-date: 2017-05-31
+:page-layout: blog
+:page-author: mhusnain
```

Comment:
Because you put mhusnain at the author you must add a file mhusmain.adoc in the _data/team folder.
Otherwise the build of the web site will fail